gluestick/singer.py

```diff
@@ -225,13 +303,21 @@ def to_singer(
         Allow or not objects to the parsed, if false defaults types to str.
 
     """
+    catalog_schema = os.environ.get("USE_CATALOG_SCHEMA", False)
```

Comment:
Shouldn't this be `os.environ.get("USE_CATALOG_SCHEMA", "false") == "true"`? Env vars are always strings

Reply:
fixed, thank you! 😃